spec/models/dimensions/date_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Dimensions::Date, type: :model do
+  let(:date) { ::Date.new(2017, 12, 21) }
```

Comment:
The date is only used in the `.exists?` `describe`. Would you mind moving it there? It is better to have the fixtures as close as possible to the test.